openfpga_flow/misc/ys_tmpl_yosys_vpr_flow_with_rewrite.ys

```diff
@@ -1,6 +1,7 @@
 # Yosys synthesis script for ${TOP_MODULE}
 # Read verilog files
 ${READ_VERILOG_FILE}
+read_verilog -lib -specify ${YOSYS_CELL_SIM_VERILOG}
```

Comment:
I am cool with this since this script is mainly used by QuickLogic's synthesis now.